### ISSUE TITLE
BackPropagation？

### ISSUE THREAD

**Issue by xwuShirley** [switchablenorms/Switchable-Normalization#7]
I am a beginner of PyTorch. Don't we need to write something like a backward function, like the last section in your paper https://arxiv.org/abs/1806.10779 (the appendix, equation(6)-(11) )? I thought you need to define backward function in the class SwitchNorm(nn.Module), or it automatically backward through the forward question?
